mobile-config.js

```diff
@@ -69,3 +69,28 @@ App.configurePlugin("cordova-fabric-plugin", {
   FABRIC_API_KEY: process.env.FABRIC_API_KEY,
   FABRIC_API_SECRET: process.env.FABRIC_API_SECRET,
 });
+
+App.appendToConfig(`
+    <universal-links>
+      <ios-team-id value="7TM8C6252Z" />
+      <preference name="AndroidLaunchMode" value="singleInstance" />
+      <host name="beta.newspring.cc" scheme="https" event="universalLinkRoute">
+        <path url="/articles" />
+        <path url="/articles/*" />
+        <path url="/devotionals" />
+        <path url="/devotionals/*" />
+        <path url="/studies" />
+        <path url="/studies/*" />
+        <path url="/studies/*/*" />
+        <path url="/stories" />
+        <path url="/stories/*" />
+        <path url="/sermons" />
+        <path url="/sermons/*" />
+        <path url="/sermons/*/*" />
+        <path url="/music/*" />
+        <path url="/locations" />
+        <path url="/watchandread" />
+      </host>
+      <host name="beta-my.newspring.cc" scheme="https" event="universalLinkRoute" />
```

Comment:
Lets also go ahead and make this production for my and newspring and do corresponding PR's that can be release when the build is live